src/PhpPresentation/Writer/PowerPoint2007/AbstractDecoratorWriter.php

```diff
@@ -51,12 +51,8 @@ protected function writeRelationship(XMLWriter $objWriter, $pId = '1', $pType =
      * @param  string                         $pElementName Element name
      * @throws \Exception
      */
-    protected function writeBorder(XMLWriter $objWriter, $pBorder, $pElementName = 'L')
+    protected function writeBorder(XMLWriter $objWriter, Border $pBorder, $pElementName = 'L')
     {
-        if (!($pBorder instanceof Border)) {
-            return;
-        }
-
         if ($pBorder->getLineStyle() == Border::LINE_NONE && $pElementName == '') {
```

Comment:
Maybe I do revert this modification cause can be null and instanceof Border check this

Same for other bellow

What do you think ?

Reply:
@tolbon Please revert this modification (and equivalent)